CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+This is an adapted version of the contributing guide used by the Community Committee: https://github.com/nodejs/community-committee/blob/master/CONTRIBUTING.md
+
+## Vocabulary
+
+* A **Contributor** is any individual creating or commenting on an issue or pull request,
+  or contributing in some other way.
+* A **Collaborator** is a contributor who has been given write access to the repository.
```

Comment:
Is it worth mentioning that they're a part of the website-redesign team?

Reply:
i suppose so, since they're intended to overlap